package.json

```diff
@@ -38,7 +38,7 @@
   "dependencies": {
     "commander": "2.12.2",
     "js-reporters": "1.2.1",
-    "resolve": "1.5.0",
+    "resolve": "^1.9.0",
```

Comment:
This swaps to using `^` whereas all of the other dependencies are specified explicitly. 

I don't really understand why we would want to pin everything the way it is currently being done. IMHO we should swap all of these to using caret, but it seems like something we should fix across the board and not per-dep...

Reply:
I'll defer to Trent, but my personal preference would be to pin in this PR and then we can unpin everything in a future PR (if the team agrees).